src/test/checkqueue_tests.cpp

```diff
@@ -139,6 +139,11 @@ struct FrozenCleanupCheck {
             cv.wait(l, []{ return nFrozen.load(std::memory_order_relaxed) == 0;});
         }
     }
+    FrozenCleanupCheck(FrozenCleanupCheck&& other) noexcept
```

Comment:
in 68d4728ff4d0fd14aa944b8d67a47c2fbecd980b: Replace `// swaps in default initialized Checks` with `// moves in default initialized Checks`?

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/26749#issuecomment-1370149202).